common/model-views.cpp

```diff
@@ -2917,6 +2921,102 @@ namespace rs2
                         }
                     }
                 }
+
+                // Draw IR reflectivity on depth frame
+                if (!viewer.skip_option(RS2_OPTION_ENABLE_IR_REFLECTIVITY))
+                {
+                    if (ds.supports(RS2_OPTION_ENABLE_IR_REFLECTIVITY) && (ds.get_option(RS2_OPTION_ENABLE_IR_REFLECTIVITY) == 1.0f))
+                    {
+                        // Add depth sample to the history
+                        viewer.ref.add_input(val, x, y);
```

Comment:
If it's a "depth sample", perhaps it's a better name than "input"?

Reply:
👍